# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424737878 28800
#      Mon Feb 23 16:31:18 2015 -0800
# Node ID 7239100e85704b58c949a9363c4f106c105650e0
# Parent  426525fb3b9f02e9bcd6ec80bb4f7ed99c8f14f4
bessel.m: Suppress useless backtrace from error() when help file called as fcn.

* bessel.m: Suppress useless backtrace from error() when help file called as fcn.

diff --git a/scripts/specfun/bessel.m b/scripts/specfun/bessel.m
--- a/scripts/specfun/bessel.m
+++ b/scripts/specfun/bessel.m
@@ -79,14 +79,14 @@
 ##
 ## @item
 ## Error---no computation, algorithm termination condition not met,
 ## return @code{NaN}.
 ## @end enumerate
 ## @end deftypefn
 
 function bessel ()
-  error ("bessel: you must use besselj, bessely, besseli, besselk, or besselh");
+  error ("bessel: you must use besselj, bessely, besseli, besselk, or besselh\n");
 endfunction
 
 
 %!error bessel ()
 
